astro-docs/src/content/docs/technologies/java/maven/introduction.mdoc

```diff
@@ -106,16 +112,8 @@ The `@nx/maven` is configured in the `plugins` array in `nx.json`.
 ```json
 // nx.json
 {
-  "plugins": [
-    {
-      "plugin": "@nx/maven",
-      "options": {
-        "testTargetName": "test",
-        "buildTargetName": "build"
-      }
-    }
-  ]
```

Comment:
are we documenting the plugin options somewhere else? or are there no more options for the maven plugin?

Reply:
There are none atm.